src/d3a/d3a_core/export.py

```diff
@@ -182,24 +182,26 @@ def _export_spot_markets_stats(self, area: Area, directory: dir, is_first: bool)
                                          is_first)
         if not area.children:
             return
-        self._export_trade_csv_files(past_markets=area.past_markets,
-                                     directory=directory,
-                                     filename=f"{area.slug}-trades",
-                                     is_first=is_first)
-
-        self._export_area_offers_bids_csv_files(past_markets=area.past_markets,
-                                                market_member="offer_history",
-                                                directory=directory,
-                                                filename=f"{area.slug}-offers",
-                                                labels=("slot",) + Offer.csv_fields(),
-                                                is_first=is_first)
-
-        self._export_area_offers_bids_csv_files(past_markets=area.past_markets,
-                                                market_member="bid_history",
-                                                directory=directory,
-                                                filename=f"{area.slug}-bids",
-                                                labels=("slot",) + Bid.csv_fields(),
-                                                is_first=is_first)
+        self._export_offers_bids_trades_to_csv_files(past_markets=area.past_markets,
```

Comment:
Nit: do you think you could rename this following the recent slack conversation about "orders" terminology? This would be something like: `_export_orders_trades_to_csv_files()` (for example).

Reply:
Not necessary of course, we could have a refactoring ticket in the future.